
/*
#pragma once

#include <QtWidgets/QMainWindow>
#include "ui_de.h"




class dehisto : public QMainWindow
{
	Q_OBJECT

public:
	dehisto(QWidget *parent = Q_NULLPTR);

private:
	Ui::deClass ui;
	

public slots:
	// de de::operator++(int n);
	void dehistor();
	void denormal();

};
*/